<!DOCTYPE HTML>
<html>

<head>
  <title>Pixu Shi's Homepage</title>
  <meta name="description" content="website description" />
  <meta name="keywords" content="website keywords, website keywords" />
  <meta http-equiv="content-type" content="text/html; charset=windows-1252" />
  <link rel="stylesheet" type="text/css" href="template/style.css" title="style" />
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of the text -->
          <h1>Pixu Shi</h1>
          <h2>Department of Biostatistics and Bioinformatics</h2>
          <h2>Duke University</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected page - to highlight which page you're on -->
          <li><a href="index.html">Home</a></li>
          <li class="selected"><a href="Pixu_publication.html">Publication</a></li>
          <li><a href="Pixu_software.html">Software</a></li>
          <li><a href="Pixu_teaching.html">Teaching</a></li>
        </ul>
      </div>
    </div>
    <div id="site_content">
      <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <!-- insert your sidebar items here -->
			<img src="Pixu_profile_square.jpg" width="195" align="middle">
          </div>
          <div class="sidebar_base"></div>
	<div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Useful Links</h3>
            <ul>
              <li><a href="https://scholars.duke.edu/person/Pixu.Shi">My profile at Duke University</a></li>
              <li><a href="https://github.com/pixushi">My GitHub Page</a></li>
              <li><a href="https://scholar.google.com/citations?user=EbF6EYMAAAAJ&hl=en">My Google Scholar Page</a></li>
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
        </div>
      </div>
      <div id="content">
		<h2>Publications</h2>
		<ul>

		<h4>Statistical methods</h4>
		


<li>Tan, J., <strong>Shi, P.</strong>, Zhang, A. R. (2024): <a href = "https://arxiv.org/abs/2410.03619">Functional Singular Value Decomposition.</a> <br /> 
             <I>arXiv</I><br />
</li>	
		
<li>Zhuang, H., Gai, X., Zhang, A. R., Hou, W., Ji, Z., and <strong>Shi, P.</strong> (2024): <a href = "https://www.biorxiv.org/content/10.1101/2024.09.14.613024">Trajectory-Guided Dimensionality Reduction for Multi-Sample Single-Cell RNA-seq Data Reveals Biologically Relevant Sample-Level Heterogeneity.</a> <br /> 
             <I>bioRxiv</I><br />
</li>

<li>Alam, M. S., Staicu, A., and <strong>Shi, P.</strong> (2024): <a href = "https://arxiv.org/pdf/2409.13819">Supervised Low-Rank Approximation of High-Dimensional Multivariate Functional Data via Tensor Decomposition.</a> <br /> 
             <I>arXiv</I><br />
</li>		

<li><strong>Shi, P. et al.</strong> (2024): <a href = "https://link.springer.com/article/10.1186/s13059-024-03453-x">Time-Informed Dimensionality Reduction for Longitudinal Microbiome Studies.</a> <br /> 
             <I>Genome Biology</I>, 25 (1), 317<br />
</li>	

<li>Han, R., <strong>Shi, P.</strong> and Zhang, A. R. (2024): <a href = "https://www.tandfonline.com/doi/abs/10.1080/01621459.2022.2153689">Guaranteed Functional Tensor Singular Value Decomposition.</a> <br /> 
             <I>Journal of the American Statistical Association</I>, 119 (546), 995-1007.<br />
</li>

    <li><strong>Shi, P.</strong>, Zhou, Y. and Zhang, A. R. (2022): <a href = "https://academic.oup.com/biomet/advance-article-abstract/doi/10.1093/biomet/asab020/6203806">High-dimensional Log-Error-in-Variable Regression with Applications to Microbial Compositional Data Analysis.</a> <br /> 
            <I>Biometrika</I>, 109(2): 405-420.<br />
    </li>
    
    <li>Lu, J. <strong>Shi, P.</strong> and Li, H. (2019): <a href = "https://onlinelibrary.wiley.com/doi/abs/10.1111/biom.12956">Generalized Linear Models with Linear Constraints for Microbiome Compositional Data.</a> <br /> 
            <I>Biometrics</I>, 75(1): 235-244.<br />
    </li>
    
    <li><strong>Shi, P.</strong> and Li, H. (2017): <a href = "http://onlinelibrary.wiley.com/doi/10.1111/biom.12681/full">A model for paired-multinomial data and its application to analysis of data on a taxonomic tree.</a> <br />
			<I>Biometrics</I>, 73(4): 1266-1278.<br />
            <a href = "https://github.com/pixushi/correlated_multinomial">Download the R codes</a>
			</li>
			
    <li><strong>Shi, P.</strong>, Zhang, A. and Li, H. (2016): <a href = "https://projecteuclid.org/euclid.aoas/1469199903">Regression analysis for microbiome compositional data.</a> <br />
			<I>Annals of Applied Statistics</I>, 10(2): 1019-1040. <br />
            <a href = "https://github.com/pixushi/CI_compositional_regression">Download the Matlab codes</a>
			</li>
			
    <li>Fogarty, C. B., <strong>Shi, P.</strong>, Mikkelsen, M. E. and Small, D. S. (2014): <a href = "http://www.tandfonline.com/doi/abs/10.1080/01621459.2016.1138865">Randomization inference and sensitivity analysis
for composite null hypotheses with binary
outcomes in matched observational studies.</a> <br />
			<I>Journal of the American Statistical Association</I>, 112(517): 321-331.
			</li>
			
			<li>Lin, W., <strong>Shi, P.</strong>, Feng, R. and Li, H. (2014): <a href = "https://academic.oup.com/biomet/article/101/4/785/1775476/Variable-selection-in-regression-with">Variable selection in regression with compositional covariates.</a> <br />
			<I>Biometrika</I>, 101(4): 785-797. <br />
            <a href = "https://github.com/pixushi/constrained_lasso">Download the R codes</a>
			</li>
			

		</ul>
		<h4>Microbiome and omics</h4>
		<ul>
		  
    <li>Kelly, M. S., <strong>Shi, P.</strong>, et al. (2024): <a href = "https://www.medrxiv.org/content/10.1101/2024.10.22.24315478">The Role of the Microbiota in Respiratory Virus-Bacterial Pathobiont Relationships in the Upper Respiratory Tract.</a> <br /> 
             <I>medRxiv</I><br />
    </li> 
    
    <li>Kim, Y. J., McQuade, V. L., <strong>Shi, P.</strong>, et al. (2024): Glucagon-like peptide 1 receptor (Glp1r) deficiency does not appreciably alter airway inflammation or gut-lung microbiome axis in a mouse model of obese allergic airways disease and bariatric surgery. <br /> 
             <I>Journal of Asthma and Allergy</I>, accepted.<br />
    </li>  
    
    <li>Hudson, A., et al., <strong>Shi, P.</strong>, and Reeves, K. (2024): Longitudinal analysis of rhesus macaque metabolome during acute SIV infection reveals disruption in broad metabolite classes. <br /> 
             <I>Journal of Virology</I>, accepted.<br />
    </li> 
    
    <li>Burcham A. M. et al. (2024): <a href = "https://www.nature.com/articles/s41564-023-01580-y">A Conserved Interdomain Microbial Network Underpins Cadaver Decomposition Despite Environmental Variables</a> <br /> 
             <I>Nature Microbiology</I>, 9 (3), 595-613 <br />
    </li> 
    
    <li>Schweickart, A. et al. (2024): <a href = "https://www.nature.com/articles/s44324-024-00016-3">Serum and CSF Metabolomics Analysis Shows Mediterranean Ketogenic Diet Mitigates Risk Factors of Alzheimer's Disease</a> <br /> 
             <I>npj Metabolic Health and Disease</I>, 2(1), 15<br />
    </li> 

    <li>SenNet Consortium (2022): <a href = "https://www.nature.com/articles/s43587-022-00326-5">NIH SenNet Consortium to map senescent cells throughout the human lifespan to understand physiological health. </a> <br />
    <I>Nature Aging</I>, 1-11. (<strong>Shi, P.</strong> among other authors as a member of
the SenNet consortium) <br />
    </li>
    
		</ul>
		<h4>Other</h4>
		<ul>
		  
			<li> <a href = "http://repository.upenn.edu/edissertations/2013/">Statistical methods for compositional and tree-structured count data in human microbiome studies</a><br />
			<I>PhD Thesis</I>, 2016.</li>
			<li> <strong>Shi, P.</strong> and Zhang, A.: <a href = "http://arxiv.org/abs/1305.4896">Methods to calculate the upper bound of Gini coefficient based on grouped data and the result for China</a><br />
			<I>preprint of Institute of Mathematics, Peking University</I>, 2010-20.</li>
		</ul>
      </div>
    </div>
    <div id="content_footer"></div>
    <div id="footer">
      Copyright &copy; textured_stars_light | <a href="http://validator.w3.org/check?uri=referer">HTML5</a> | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a> | <a href="http://www.html5webtemplates.co.uk">design from HTML5webtemplates.co.uk</a>
    </div>
  </div>
<!-- Start of StatCounter Code for Default Guide -->
<script type="text/javascript">
var sc_project=11414960;
var sc_invisible=1;
var sc_security="e117e79e";
var scJsHost = (("https:" == document.location.protocol) ?
"https://secure." : "http://www.");
document.write("<sc"+"ript type='text/javascript' src='" + scJsHost+
"statcounter.com/counter/counter.js'></"+"script>");
</script>
<noscript><div class="statcounter"><a title="web analytics"
href="http://statcounter.com/" target="_blank"><img class="statcounter"
src="//c.statcounter.com/11414960/0/e117e79e/1/" alt="web
analytics"></a></div></noscript>
<!-- End of StatCounter Code for Default Guide -->
</body>
</html>
